virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -182,6 +182,13 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 [id="virt-4-15-bug-fixes"]
 == Bug fixes
+* Previously, EFI pipelines failed when started with a storage class that had the `volumeBindingMode` set to `WaitForFirstConsumer`. This fix removes the annotation in the `StorageClass` object that was causing the pipelines to fail. (link:https://issues.redhat.com/browse/CNV-32287[*CNV-32287*])
+
+* Previously, if you simultaneously cloned more than 1000 VMs using the provided data sources in the `openshift-virtualization-os-images` namespace, not all of the VMs moved to a running state. With this fix, you can clone a large number of VMs concurrently. (link:https://issues.redhat.com/browse/CNV-30083[*CNV-30083*])
+
+* Previously, you could not SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. With this update, you can use a `NodePort` service for VM SSH access. (link:https://issues.redhat.com/browse/CNV-24889[*CNV-24889*])
```

Comment:
```suggestion
* Previously, you could not SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. With this update, you can use a `NodePort` service for VM SSH access. (link:https://issues.redhat.com/browse/CNV-24889[*CNV-24889*])
```
@phoracek I think this should be revised to refer to GUI and ease of use.

Reply:
@phoracek Do you have any suggestions to update the wording for the above issue?

Reply:
I added a suggestion for an alternative text.